src/core/objective.jl

```diff
@@ -133,12 +140,19 @@ function objective_owf(wm::AbstractWaterModel)::JuMP.AffExpr
     end
 
     for n in network_ids_flow
+        println("Warning: Changing pump and ne_pump objectives to min power")
         for a in ids(wm, n, :pump)
             # Add pump energy costs to the objective.
-            JuMP.add_to_expression!(objective, var(wm, n, :c_pump, a))
+            JuMP.add_to_expression!(objective, var(wm, n, :P_pump, a))
```

Comment:
I don't think we should change the objective to minimize power instead of cost, here. The standard OWF minimizes cost. If you want another objective that minimizes power, maybe add this as `objective_power` or something similar?